package.json

```diff
@@ -7,7 +7,7 @@
 	"author": {
 		"name": "Sindre Sorhus",
 		"email": "sindresorhus@gmail.com",
-		"url": "sindresorhus.com"
+		"url": "https://sindresorhus.com"
```

Comment:
Don't do unrelated changes.

Reply:
This was to satisfy my editor haha. I reverted it. :)